web/src/main/webapp/WEB-INF/classes/org/fao/geonet/api/Messages_fre.properties

```diff
@@ -190,6 +190,8 @@ exception.resourceInvalid.metadata=Les fiches ne sont pas valides
 exception.resourceInvalid.metadata.description=Les fiches ne peuvent pas \u00EAtre soumises ou approuv\u00E9es
 exception.resourceNotEnabled.workflow=Le workflow est d\u00E9sactiv\u00E9
 exception.resourceNotEnabled.workflow.description=Impossible de d\u00E9finir l'\u00E9tat des fiches
+exception.resourceNotEnabled.groupWorkflow=Le workflow de groupe ''{0}'' est d\u00E9sactiv\u00E9
+exception.resourceNotEnabled.groupWorkflow.description=Impossible de d\u00E9finir l'\u00E9tat des fiches dans le groupe ''{0}''
```

Comment:
Can you rebase the pull request and update `Messages_dut.properties`, including for now the English values?